figures/scripts/supp-subtype-umap.R

```diff
@@ -0,0 +1,212 @@
+# Run Jin and Jo Lynne Rokita 
+#
+# Generate figures with UMAP results 
+
+### Load libraries
+library(ggplot2)
```

Comment:
We don't need to also load `ggplot2` if `tidyverse` is being loaded. 
```suggestion
```

Reply:
Removed accordingly.